lib/ControlledVocab/ControlledVocab.js

```diff
@@ -132,6 +136,23 @@ class ControlledVocab extends React.Component {
       connect: PropTypes.func.isRequired,
       hasPerm: PropTypes.func.isRequired
     }).isRequired,
+    /*
+      sample translation prop
+      {
+        cannotDeleteTermHeader: "Cannot delete {type}",
+        cannotDeleteTermMessage: "This {type} cannot be deleted, as it is in use by one or more records.",
+        deleteEntry: "Delete {entry}",
+        termDeleted: "The {type} <b>{term}</b> was successfully <b>deleted</b>",
+        termWillBeDeleted: "The {type} <b>{term}</b> will be <b>deleted.</b>"
```

Comment:
This is a great start, but it's confusing because these examples use placeholders for `type` and `term`, but `{type}` won't be a variable in this implementation (e.g. for [`cannotDeleteTermHeader`](https://github.com/folio-org/stripes-smart-components/blob/666c28b204dcbe402bcacc5e7a80e0cb5a98e03d/lib/ControlledVocab/ControlledVocab.js#L393)) while `{term}` still is (e.g.  for [`termDeleted`](https://github.com/folio-org/stripes-smart-components/blob/666c28b204dcbe402bcacc5e7a80e0cb5a98e03d/lib/ControlledVocab/ControlledVocab.js#L331-L336)). I think we need something like this:
```
cannotDeleteTermHeader: "Cannot delete patron group",
cannotDeleteTermMessage: "This patron group cannot be deleted, as it is in use by one or more records.",
deleteEntry: "Delete patron group",
termDeleted: "The patron group <b>{term}</b> was successfully <b>deleted</b>",
termWillBeDeleted: "The patron group <b>{term}</b> will be <b>deleted.</b>"
```
Sorry I wasn't more specific in my initial comment. I clearly didn't think through the details until now.